src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenSystemNullableTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+{
+    public class CodeGenSystemNullableTests : CSharpTestBase
+    {
+        [Fact]
+        [WorkItem(65091, "https://github.com/dotnet/roslyn/issues/65091")]
+        public void Nullable_Parameter_EqualsNull()
+        {
+            var source = @"
+void M(in int? v)
+{
+    if (v == null) 
+        return;
+}";
+            var verifier = CompileAndVerify(source);
```

Comment:
>CompileAndVerify(source) [](http://example.com/codeflow?start=27&length=24)

Please update the new passing tests to also check execution (`CompileAndVerify(..., expectedOutput: ...)`) using `Console.Write(...)` in the source of the compilation.

Reply:
Also, since the three tests are equivalent (same IL), consider consolidating into one with a `[Theory]`, using something like `[InlineData("v == null")][InlineData("!v.HasValue")]...`